README.md

```diff
@@ -37,24 +37,36 @@ Will result into this HTML (open graph metas are using property automatically) :
 
 Title & metas are set only on `didTransition` in case the route would be canceled.
 
+**Note**: og tags will not correctly work without using FastBoot or some sort of server side rendering.
```

Comment:
@j15e I was talking about what you would use og tags for, like posting to facebook, twitter, etc. I was unaware Google even look at the og tags, thought they had all their own metas they looked at.